src/server/routes/adminPanel.tsx

```diff
@@ -52,8 +52,9 @@ router.get('/', auth.isAuthenticated, auth.isAuthorized(ADMIN), async (req, res,
 	const {orm} = req.app.locals;
 	const query = parseQuery(req.url).get('q') ?? '';
 	const type = 'editor';
-	const size = req.query.size ? parseInt(parseQuery(req.url).get('size'), 10) : 20;
-	const from = req.query.from ? parseInt(parseQuery(req.url).get('from'), 10) : 0;
+	const urlQry = parseQuery(req.url);
+	const size = getIntFromQueryParams(urlQry, 'size', 20);
+	const from = getIntFromQueryParams(urlQry, 'from');
```

Comment:
```suggestion
	const size = getIntFromQueryParams(urlQuery, 'size', 20);
	const from = getIntFromQueryParams(urlQuery, 'from');
```
See my comment above. Let's be generous and grant the variable name two more letters as well :grinning:

Reply:
haha! fixed.